playbooks/eap8.yml

```diff
@@ -32,6 +30,11 @@
        file: 'molecule.*yml$'
      - match: "eap_version: 7.4.0"
        replace: "eap_version: 8.0.0"
+     - match: "eap_apply_cp: [tT]rue"
+       replace: "eap_apply_cp: false"
```

Comment:
@guidograzioli For performance purpose, we might want to identify the 2 or 3 files this is need, and have three replacement with the file: attribute?